misc/python/materialize/checks/backup_actions.py

```diff
@@ -0,0 +1,68 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Use of this software is governed by the Business Source License
+# included in the LICENSE file at the root of this repository.
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0.
+
+from typing import TYPE_CHECKING
+
+from materialize.checks.actions import Action
+from materialize.checks.executors import Executor
+from materialize.mzcompose.services.minio import MINIO_BLOB_URI
+
+if TYPE_CHECKING:
```

Comment:
Why do we need this?

Reply:
copy and paste from another file that needed this to satisfy the mypy type checker -- without it the type checker could not type check and python complained about a circular dependency.